<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Character_edit</class>
 <widget class="QDialog" name="Character_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Character_table" name="tv_characters">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>391</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>55</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_add">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_edit">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_delete">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Редактор характеристик</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Character_table</class>
   <extends>QTableWidget</extends>
   <header>character/character_table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
